packages/chemical-elements/0.1.0/README.md

```diff
@@ -0,0 +1,3 @@
+A package containing all currently known chemical elements and its abreviations.
+Just type the chemical symbol preceded by semicolon and it will be replaced by its name.
```

Comment:
```suggestion
Just type the chemical symbol preceded by a colon and it will be replaced by its name.
```

Or, should alternatively all the triggers be updated to use `;`?

Reply:
colon is the correct word